windows/hid.c

```diff
@@ -479,22 +546,6 @@ static void hid_internal_get_info(const wchar_t* interface_path, struct hid_devi
 	if (cr != CR_SUCCESS)
 		goto end;
 
-	/* Get the hardware ids from devnode */
-	hardware_ids = hid_internal_get_devnode_property(dev_node, &DEVPKEY_Device_HardwareIds, DEVPROP_TYPE_STRING_LIST);
-	if (!hardware_ids)
-		goto end;
-
-	/* Search for interface number in hardware ids */
-	for (wchar_t* hardware_id = hardware_ids; *hardware_id; hardware_id += wcslen(hardware_id) + 1) {
-		/* Normalize to upper case */
-		for (wchar_t* p = hardware_id; *p; ++p) *p = towupper(*p);
-
-		dev->interface_number = hid_internal_get_interface_number(hardware_id);
```

Comment:
this has a side-effect: `interface_number` is not set to `-1` for non-USB devices anymore
we probably need to make `hid_internal_get_device_info` set it to `-1` by default

Reply:
actually seems this -1 is not documented properly in `hidapi.h`